docker/dev/site.conf

```diff
@@ -17,6 +17,10 @@ server {
     rewrite ^/api/(?:v1|test)/?$ /wikia.php?controller=ApiDocs&method=index break;
     rewrite ^/api/(?:v1|test)/([^/]*)/([^/]*) /wikia.php?controller=$1Api&method=get$2 break;
 
+    # SUS-5798 / SUS-5824: alternative article paths - /wiki/index.php and /w
+    rewrite ^/w/(.*)$ http://$host/wiki/$1 permanent;
```

Comment:
This should probably also work for new language paths

Reply:
Handled in 44f70c1